wondrous-app/components/Settings/TaskImport/GenericImportTaskModal/styles.tsx

```diff
@@ -0,0 +1,290 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  CircularProgress,
+  MenuItem,
+  Modal,
+  Select,
+  Typography,
+} from '@mui/material';
+import styled from 'styled-components';
+import CloseModalIcon from 'components/Icons/closeModal';
+import AttachFileSvg from 'components/Icons/attachFile.svg';
+import { Button } from 'components/Common/button';
+import typography from 'theme/typography';
+import palette from 'theme/palette';
+
+export const GenericImportTaskModalWrapper = styled(Modal)`
```

Comment:
Can you use `components/Modal` ? You can check the UI in the storybook by running `yarn storybook`

Reply:
![screen-2022-10-08-23-32-13-buyug](https://user-images.githubusercontent.com/1454782/194728462-76fc5e56-367a-48b7-bc02-54441a5d67b0.jpg)

Reply:
done